LCM/dsc/engine/EngineHelper/EventWrapper.c

```diff
@@ -25,8 +25,10 @@ ConfigurationDetails g_ConfigurationDetails = { 0, 0, MI_FALSE };
 #else
 #include <pal/cpu.h>
 ConfigurationDetails g_ConfigurationDetails;
-static FILE *_DSCos;
-static Log_Level _DSClevel = OMI_WARNING;
+static FILE *_DSCLogFile;
+static FILE *_DSCDetailedLogFile;
+static Log_Level _DSCLogLevel = OMI_WARNING;
+static Log_Level _DSCDetailedLogLevel = OMI_VERBOSE;
```

Comment:
1- In current DSC implementation where does debug logs go ? Do they go to same dsc.log/omsconfig.log file ? 

2- how are debug logs enabled for DSC. 

3- On customer machine are debug logs disabled by default ?

Reply:
more questions.

Are we planning to add new logs or this change is to split existing logs in two files ?

Reply:
Can you please comment on motivation of this change ? Does omsagent & omi follow same practice where debug and other logs are separated in two different files @lagalbra can you comment on this ?

Reply:
1- In current DSC implementation where does debug logs go ? Do they go to same dsc.log/omsconfig.log file ? 
[Amit] debug/warning/error logs goes to dsc.log/omsconfig.log  file.
2- how are debug logs enabled for DSC. 
event wrapper filters the logs, if it is error/debug/warning it writes in the log file otherwise skips. there is no mechanism to enable/disable
3- On customer machine are debug logs disabled by default ? 

---
In reply to: [140114396](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/327#discussion_r140114396) [](ancestors = 140114396)